src/pudl/transform/ferc1.py

```diff
@@ -806,6 +809,37 @@ def transform(
         return df
 
 
+class PlantsHydroFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """A table transformer specific to the ``plants_hydro_ferc1`` table."""
+
+    table_id: Ferc1TableId = Ferc1TableId.PLANTS_HYDRO_FERC1
+
+    @cache_df(key="main")
+    def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Table specific transforms for plants_hydro_ferc1. Hi!
+
+        Params:
+            df: Pre-processed, concatenated XBRL and DBF data.
+
+        Returns:
+            A single transformed table concatenating multiple years of cleaned data
+            derived from the raw DBF and/or XBRL inputs.
+        """
+        df = (
+            self.normalize_strings(df)
+            .pipe(self.categorize_strings)
+            .pipe(self.convert_units)
+            .pipe(self.nullify_outliers)
+            .pipe(self.drop_invalid_rows)
+        )
+        df["project_num"] = self.strip_col_of_non_ints(df["project_num"])
+        return df
+
+    def strip_col_of_non_ints(self, col: pd.Series) -> pd.DataFrame:
+        """Strip the column of any non-integer values."""
+        return col.str.extract(r"(\d+)")
```

Comment:
hm yes this is probably true. This feels less overall useful to me for some reason than `enforce_snake_case`. My inclination is to wait to see if it is more generally applicable and if so we can move it over np.

Reply:
This could get weird if you have a floating point or exponential number, or some other gobbledygook string number mashup.  Like do you intend for 1.00 to be interpreted as 100? You might try `pd.to_numeric(col).astype(int)` or something similar.

Reply:
i already changed it to use `to_numeric` with `errors="coerce"`

Reply:
@aesharpe  you were right about this suggestion for sure! i jumped into trying to transform the pumped storage table  and this same treatment is required over there... I'm going to migrate this in that table's pr instead of here so i don't have weird merge issues if that's alright but i wanted to let you know that i probably should have listened to you in the first place!